pydra/engine/tests/test_shelltask.py

```diff
@@ -2758,30 +2760,38 @@ def test_shell_cmd_outputspec_2a_exception(plugin, tmpdir):
 
 
 @pytest.mark.parametrize("results_function", [result_no_submitter, result_submitter])
-def test_shell_cmd_outputspec_3(plugin, results_function, tmpdir):
+def test_shell_cmd_outputspec_3(plugin, results_function, tmp_path):
     """
     customised output_spec, adding files to the output,
     using a wildcard in default, should collect two files
     """
     cmd = ["touch", "newfile_tmp1.txt", "newfile_tmp2.txt"]
     my_output_spec = SpecInfo(
         name="Output",
-        fields=[("newfile", File, "newfile_*.txt")],
+        fields=[("newfile", MultiOutputFile, "newfile_*.txt")],
         bases=(ShellOutSpec,),
     )
     shelly = ShellCommandTask(
-        name="shelly", executable=cmd, output_spec=my_output_spec, cache_dir=tmpdir
+        name="shelly", executable=cmd, output_spec=my_output_spec, cache_dir=tmp_path
     )
 
     res = results_function(shelly, plugin)
     assert res.output.stdout == ""
     # newfile is a list
     assert len(res.output.newfile) == 2
-    assert all([file.exists for file in res.output.newfile])
+    assert all([file.fspath.exists() for file in res.output.newfile])
 
 
+@pytest.mark.xfail(
+    reason=(
+        "This test doesn't look like it ever worked properly. The command isn't being "
+        "split on ';' and instead the arguments are just treated as a list of dirs to create. "
+        "This includes 'tmp/newfile.txt', which fileformats now correctly detects as being "
+        "a directory instead of a file"
```

Comment:
If this had happened to work for a time, I suppose that's fine, but it doesn't seem worth testing and supporting. I vote for just removing these tests.